src/strategies/safe-vested/examples.json

```diff
@@ -0,0 +1,24 @@
+[
+    {
+      "name": "Example query",
+      "strategy": {
+        "name": "safe-vested",
+        "params": {
+            "allocationsSource": "https://raw.githubusercontent.com/5afe/claiming-app-data/main/resources/data/snapshot-allocations-test.json"
```

Comment:
This could be edited in the future right? To avoid problems I would suggest using a IPFS file or a file with a commit id like this https://raw.githubusercontent.com/5afe/claiming-app-data/7942c2b6757c02fadb32d58dde09a12a35f5a462/resources/data/snapshot-allocations-test.jsonso the voting power remain same per proposal

Reply:
I added the fixed commit for the example link

Thank you for your feedback!